### PR TITLE
Support for Polish OLED characters

### PR DESCRIPTION
**Pull request by And137**
Added support for Polish OLED characters.

- Custom FONT_SMALL ArialMT_Plain_10_PL

- Automatic selection between Polish and Ukrainian/Russian characters mapping depending on the -D OLED_{LANG_NAME} flag

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -2,8 +2,9 @@
...
     "recommendations": [
-		"ms-vscode.cpptools",
-        "platformio.platformio-ide",
-        "trunk.io"
+        "platformio.platformio-ide"
     ],
+    "unwantedRecommendations": [
+        "ms-vscode.cpptools-extension-pack"
+    ]
 }
```
Comment (caveman99) [inline .vscode/extensions.json:10]: please add the vscode files to your ignore-on-commit list.

**Review by caveman99 — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json (hunk 1 above)
Comment (And137) [reply .vscode/extensions.json:10]: Ok :D

**Comment by And137** [comment]
Ready :D

**Comment by And137** [comment]
For some reason the license/CLA is still pending even though I’ve already accepted the agreement.

**Comment by And137** [comment]
meshpl

**Comment by And137** [comment]
MeshPL

**Comment by And137** [comment]
MeshPL

**Review by caveman99 — APPROVED** [review]

**Comment by caveman99** [comment]
@And137 thanks, looking good. I'll send a formatting commit later today to get rid of the trunking errors. After all checks are green i'll merge it.

**Comment by And137** [comment]
Thank you very much! :D

**Pull request merged**